lib/login_screen.dart

```diff
@@ -22,31 +22,46 @@ class _login_screenState extends State<login_screen> {
             builder: (context) => Home_screen(),
           ));
     } else
-      print("please chech your pass or phoneNum");
+      ScaffoldMessenger.of(context).showSnackBar(
+        SnackBar(
+          content: Text("Your phone number or pass is incorrect"),
+        ),
+      );
   }
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
         body: SafeArea(
       child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
         children: [
+          Image.asset("assets\images\login_logo.png"),
```

Comment:
please change this path to **assets/images/login_logo.png**

Reply:
@ShamsMoAhmed

Reply:
@GenuisChip  ok, Thanks